test/unit_tests/00_itsalive/aie.mlir

```diff
@@ -8,7 +8,6 @@
 //
 //===----------------------------------------------------------------------===//
 
-// REQUIRES: valid_xchess_license
```

Comment:
The requirement here depends on the default compiler set for aiecc.py.   I don't think we can/should just remove this.

Reply:
@jackl-xilinx requested this change.  I will leave to him to elaborate.

Reply:
Thinking about this further, maybe it is reasonable, since I've been arguing that we do this check at configuration time, anyway.

Reply:
@stephenneuendorffer This check doesn't work when we run unit tests on the board with a cross compiled build since it will have peano but not xchesscc. The main issue to resolve is what the publicly built llvm (not peano) does in conjunction with aiecc.py since aiecc.py seems to require opt/llc/clang with some arch=aie options which I had previously assumed the non-peano llvm provides. Regardless, we need a different way to "disable" these tests for CI if we can't run it properly without both peano and xchesscc.